lib/common/CommonMin.h

```diff
@@ -49,9 +54,8 @@ using namespace Memory;
 #include "DataStructures/SList.h"
 #include "DataStructures/Dlist.h"
 #include "DataStructures/KeyValuePair.h"
-#include "DataStructures/DictionaryEntry.h"
-#include "DataStructures/DefaultContainerLockPolicy.h"
```

Comment:
> in this case it may have just been to be more consistent with the master branch, and not to address any particular ordering issue

I see. ```BaseDictionary``` depends on ```DefaultContainerLockPolicy```. Maybe it makes sense to have ```DefaultContainerLockPolicy.h``` stay here, then you don't need the forward declaration and change to ```runtime.h```?